util/MultiplayerCommon.cpp

```diff
@@ -34,6 +34,7 @@ namespace {
         db.Add("save.format.binary.enabled",                UserStringNop("OPTIONS_DB_BINARY_SERIALIZATION"),   false);
         db.Add("save.format.xml.zlib.enabled",              UserStringNop("OPTIONS_DB_XML_ZLIB_SERIALIZATION"), true);
         db.Add("save.auto.hostless.enabled",                UserStringNop("OPTIONS_DB_AUTOSAVE_HOSTLESS"),      true);
+        db.Add<int>("save.auto.timeout",           UserStringNop("OPTIONS_DB_AUTOSAVE_TIMEOUT"), 0, Validator<int>());
```

Comment:
indent to match

Reply:
"timeout" is misleading... maybe "interval"?

Reply:
Renamed.